docs/source/futures.rst

```diff
@@ -719,16 +722,13 @@ All operations that require talking to the remote worker are awaitable
 
        n = await counter.n  # attribute access also must be awaited
 
-
-
-
 Usually Dask computations are composed of tasks that build off of each other in
-a pure functional way.  They're centrally manathat are managed by the central
+a pure functional way.  **They're centrally manathat are managed by the central
```

Comment:
This seems like a typo in the original?

Reply:
Ah, right.  I see that you mentioned this in the original comment.  I recommend that we remove these two paragraph snippets entirely.

Reply:
If it doesn't add any useful information then it's best to remove them. I'll do it right away.